src/worker/props.js

```diff
@@ -67,7 +67,8 @@ export async function getPageProps(event) {
   }
 
   const cacheOption =
-    staticOptions && staticOptions.cache && staticOptions.cache.api
+    (staticOptions && staticOptions.cache && staticOptions.cache.api) ||
+    cache.api
```

Comment:
```suggestion
    ((staticOptions && staticOptions.cache ||  cache ) || {}).api
```

Trying to avoid `0 || xyz`